apps/platforms/linux/terminator/terminator-linux.py

```diff
@@ -0,0 +1,82 @@
+from talon import Context, actions, Module
```

Comment:
Naming the file just `terminator.py` would probably be fine.

Reply:
It currently only triggers for Linux, because I have no experience with terminator on any other operating system. I was concerned that the shortcuts could be different, and so I wanted  to make it clear but this only is known to operate on Linux. I'm not one hundred percent sure, but I believe then might be a port of this program for mac.